content/en/docs/languages/java/configuration.md

```diff
@@ -0,0 +1,405 @@
+---
+title: Configuration
+linkTitle: Configuration
+weight: 10
+aliases: [config]
+# prettier-ignore
+cSpell:ignore: authservice blrp Dotel ignore LOWMEMORY myservice ottrace PKCS retryable
+---
+
+The OpenTelemetry SDK provides a working implementation of the API, and can be
+set up and configured in a number of ways. The Java SDK supports most of the
+available [configuration options](/docs/languages/sdk-configuration/). For
+conformance details, see the
+[compliance matrix](https://github.com/open-telemetry/opentelemetry-specification/blob/main/spec-compliance-matrix.md).
+
+{{% alert title="System Properties and Environment Variables" color="info" %}}
+Any setting configurable with a system property can also be configured with an
+environment variable. Apply the following steps to convert a system property to
+an environment variable:
+
+- Convert the name to uppercase.
+- Replace all `.` and `-` characters with `_`.
+
+For example `otel.instrumentation.common.default-enabled` would convert to
+`OTEL_INSTRUMENTATION_COMMON_DEFAULT_ENABLED`.
+
+{{% /alert %}}
+
+## General
+
+The autoconfigure module registers Java shutdown hooks to shut down the SDK when
+appropriate. Please note that since this project uses `java.util.logging` for
+all of it's logging, some of that logging may be suppressed during shutdown
+hooks. This is a bug in the JDK itself, and not something we can control. If you
+require logging during shutdown hooks, please consider using `System.out` rather
+than a logging framework that might shut itself down in a shutdown hook, thus
+suppressing your log messages. See this
+[JDK bug](https://bugs.openjdk.java.net/browse/JDK-8161253) for more details.
+
```

Comment:
I recommend putting in another note about how signal refers to traces, metrics, logs:

```
{{% alert title="Signal configuration" color="info" %}}

The text  placeholder`{signal}` refers to the supported [OpenTelemetry Signal](/docs/concepts/signals/).
Valid values include `traces`, `metrics`, and `logs`.

{{% /alert %}}
```

That way folks don't think you need to specify it as `otel.{traces}.exporter` and the like.